### ISSUE TITLE
reticulate_1.18 run Chinese character string garbled

### ISSUE THREAD

**Issue by limao1995** [rstudio/reticulate#985]
Today I updated Rstudio to 1.4.1106, and also updated the reticulate package to the latest version 1.18 (the original Rstduio version is 1.3.1093, and the reticulate version is 1.16). It turns out that after loading the new version of the reticulate package, run the Python code console The output result of "Chinese character string" is garbled. 
![reticulate的bug报告](https://user-images.githubusercontent.com/32638116/114645959-f4f81700-9d0c-11eb-80d0-a44f661e0688.png)

**Comment by limao1995** [comment]
When I use the old version of Rstudio(version 1.3.1093) and reticulate(version 1.16), the above problem does not appear.

**Comment by kevinushey** [comment]
What if you use the development version? `remotes::install_github("rstudio/reticulate")`

**Comment by limao1995** [comment]
> 
> 
> What if you use the development version? `remotes::install_github("rstudio/reticulate")`

Yes, I have tried the development version on both the new and old versions of Rstudio (1.3.1093 and 1.4.1106), but the problem remains the same
![reticulate报错](https://user-images.githubusercontent.com/32638116/114799445-300c5000-9dca-11eb-93f2-08aada8f2114.png)
